packages/imperative/src/cmd/src/profiles/CommandProfiles.ts

```diff
@@ -15,8 +15,7 @@ import { ImperativeExpect } from "../../../expect";
 
 /**
  * Profiles map created by the command profile loader and passed to the handler via parameters. Handlers can
- * retreive loaded profiles from the map via the profile type.
- * @export
+ * retrieve loaded profiles from the map via the profile type.
  * @class CommandProfiles
  */
 export class CommandProfiles {
```

Comment:
Are the `CommandProfiles` and `CommandProfileLoader` classes still needed? I thought they only existed for the purpose of working with v1 profiles.

Reply:
It might be possible to remove them. However, CommandProfileLoader is called by the CommandProcessor. CommandProfileLoader implies that it gathers profile information from the command's definition, which might still be relevant. I was afraid to remove it. I did mark CommandProfileLoader as @internal making it possible for us to remove it in the future without breaking consumers.

CommandProfileLoader references CommandProfiles. CommandProfiles are also referenced by numerous interfaces. Some of those interfaces are outside of the imperative package, so I could not mark CommandProfiles as @internal.

I think that the removal of these two items may require a lot change and a lot time to correct. I was not convinced that the extra baggage was worth the extra time needed to remove it. In our next V1 profiles story, after completing its primary objectives, we could decide as a team if we want to take some extra time to try to remove these items during that story. I will make a note to revisit this subject at that time.